src/main/webapp/i18n/de/course.json

```
@@ -92,6 +92,7 @@
                 "ltiKey": "LTI-Key",
                 "ltiSecret": "LTI-Geheimnis",
                 "userPrefix": "Nutzer-Präfix",
+                "requireExistingUser": "Vorhandener Nutzer:innen erforderlich",
```

```suggestion
                "requireExistingUser": "Vorhandene:r Nutzer:in erforderlich",
```
I think you meant that